.github/workflows/test-molecule.yml

```diff
@@ -20,6 +20,9 @@ jobs:
           {role: "manage-service", test: "ssv-prysm", delay: 60},
           {role: "manage-service", test: "monitor-lighthouse", delay: 70},
           {role: "manage-service", test: "monitor-nimbus", delay: 80},
+          {role: "validator-import-api", test: "lighthouse", delay: 25},
+          {role: "validator-import-api", test: "nimbus", delay: 25},
+          {role: "validator-import-api", test: "prysm", delay: 25},
```

Comment:
I'd suggest using delays `95`, `100`, `105` for the 3 new tests to prevent vm creation errors with Hetzner.

Reply:
the delays changed to "lighthouse-95", "nimbus-100" and "prysm-105"